tm-bench/transacter.go

```diff
@@ -34,9 +34,10 @@ type transacter struct {
 	Connections       int
 	BroadcastTxMethod string
 
-	conns   []*websocket.Conn
-	wg      sync.WaitGroup
-	stopped bool
+	conns      []*websocket.Conn
+	connStatus []error
```

Comment:
maybe `errs`?

Reply:
renamed to connsBroken, since I realized the way I was doing it didn't require an error object, but a bool instead.